source-code/website/src/pages/index/index.page.tsx

```diff
@@ -25,6 +25,23 @@ export function Page() {
 					<ConfigPage />
 					<Pricing />
 					<GetStarted />
+
+					{/* Temporary Product Hunt Badge –– Will be removed after launch */}
+					<div class="max-w-screen-xl px-10 w-full sticky bottom-0 flex justify-end h-0 -translate-y-20 z-50 mx-auto pointer-events-none">
+						<a
+							href="https://www.producthunt.com/posts/inlang-vscode-extension-for-i18n-beta?utm_source=badge-featured&utm_medium=badge&utm_souce=badge-inlang&#0045;vscode&#0045;extension&#0045;for&#0045;i18n&#0045;beta"
+							target="_blank"
+							class="pointer-events-auto"
+						>
+							<img
+								src="https://api.producthunt.com/widgets/embed-image/v1/featured.svg?post_id=404697&theme=neutral"
+								alt="inlang&#0032;VSCode&#0032;extension&#0032;for&#0032;i18n&#0032;&#0040;beta&#0041; - Extension&#0032;for&#0032;i18n&#0032;codebases&#0032;making&#0032;developers&#0032;life&#0039;s&#0032;easier | Product Hunt"
+								style={{ width: "250px", height: "54px" }}
+								width="250"
+								height="54"
+							/>
+						</a>
+					</div>
```

Comment:
@floriandwt As already stated in Discord, I think it doesn't have to be sticky and can live beneath the image in the headline section – what do you think?

<img width="200" alt="Bildschirmfoto_2023-07-17_um_19 33 59" src="https://github.com/inlang/inlang/assets/34959078/84d71fe9-6e67-4c7e-a1e3-0723e6a563ab">